evil-surround.el

```diff
@@ -65,8 +65,11 @@ Each item is of the form (TRIGGER . (LEFT . RIGHT)), all strings.
 Alternatively, a function can be put in place of (LEFT . RIGHT).
 This only affects inserting pairs, not deleting or changing them."
   :group 'surround
-  :type '(repeat (cons (regexp :tag "Key")
-                       (symbol :tag "Surround pair"))))
+  :type '(alist
```

Comment:
no. I know that this pull is supposed to improve the customize interface so that the user is allowed to customize/add his own evil-surround-pairs-alist. I just want to understand how you are doing it. 

For starters, emacs does not tell me what `alist` is through `C-h f`. I find `repeat`, but no `alist`. 

Can you explain to me what exactly you are doing here and why it works? 

Thanks.